phase1-cli/src/lib.rs

```diff
@@ -82,6 +82,8 @@ pub struct Phase1Opts {
         parse(try_from_str = "subgroup_check_mode_from_str")
     )]
     pub subgroup_check_mode: SubgroupCheckMode,
+    #[options(help = "perform ratio check", default = "false")]
+    pub ratio_check: bool,
```

Comment:
I actually think we should make it the default and change this to `skip_ratio_check`